source/core/ib_particle.cc

```diff
@@ -131,24 +131,25 @@ IBParticle<dim>::get_properties()
   properties[1]  = radius * 2.0;
   properties[2]  = velocity[0];
   properties[3]  = velocity[1];
-  properties[5]  = fluid_forces[0];
-  properties[6]  = fluid_forces[1];
-  properties[8]  = omega[0];
-  properties[9]  = omega[1];
-  properties[10] = omega[2];
-  properties[11] = mass;
-  properties[12] = fluid_torque[0];
-  properties[13] = fluid_torque[1];
-  properties[14] = fluid_torque[2];
+  properties[5]  = omega[0];
+  properties[6]  = omega[1];
+  properties[7]  = omega[2];
+  properties[8]  = fluid_forces[0];
+  properties[9]  = fluid_forces[1];
+  properties[11] = fluid_torque[0];
+  properties[12] = fluid_torque[1];
+  properties[13] = fluid_torque[2];
+  properties[14] = mass;
+
   if (dim == 2)
     {
-      properties[4] = 0;
-      properties[7] = 0;
+      properties[4]  = 0;
+      properties[10] = 0;
     }
   if (dim == 3)
     {
-      properties[4] = velocity[2];
-      properties[7] = fluid_forces[2];
+      properties[4]  = velocity[2];
+      properties[10] = fluid_forces[2];
```

Comment:
It's a bad idea to use the properties "hardcoded number".
Use the enum directly like it is used everywhere else. Otherwise I have no clue what will happen down the 
drain. I know it was most likley already like this but this was a bad choice at the time. Now let's correct it :).